proposals/ssss-distributed-actor-runtime.md

```diff
@@ -920,10 +920,10 @@ public protocol DistributedTargetInvocationEncoder {
 
 
 ```swift
-public protocol DistributedTargetInvocationDecoder: AnyObject {
```

Comment:
Don’t forget to add inout to executeDistributedTarget

Reply:
thanks done!